adam_zadanie7/src/test/java/zadanie7/test/LoginTest.java

```diff
@@ -0,0 +1,22 @@
+package zadanie7.test;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import zadanie7.page.LoginPage;
+import zadanie7.page.Register;
+
+public class LoginTest extends BeforeAfterTest {
+    @BeforeMethod
+    public void Start() {
+        new LoginPage(driver, "http://parabank.parasoft.com/")
```

Comment:
Ja osobiscie napisalbym to wszystko w tescie - logowanie jest jakas akcja, ktora bezposrednio wplywa na wynik testu., czyli:
```
new LoginPage(driver, "http://parabank.parasoft.com/")
                .setLogin("adamm")
                .setPassword("1234")
                .clickLoginButton()
                .assertIfUserIsLogin("Welcome Adam M");
```